<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>everyday_sentence</class>
 <widget class="QWidget" name="everyday_sentence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lb_which_do_you_like">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>你喜欢哪种类型的风格呢</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAll">
     <property name="text">
      <string>全选</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbEmotion">
       <property name="text">
        <string>情感</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSaying">
       <property name="text">
        <string>名言</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMovie">
       <property name="text">
        <string>电影</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbInspire">
       <property name="text">
        <string>励志</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_skip">
       <property name="text">
        <string>跳过</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
